src/ert/gui/ertwidgets/analysismodulevariablespanel.py

```diff
@@ -170,8 +182,13 @@ def createDoubleSpinBox(
         )
         return spinner
 
-    def valueChanged(self, variable_name, variable_type, variable_control):
-        value = None
+    def valueChanged(
+        self,
+        variable_name: str,
+        variable_type: Type[bool] | Type[float],
+        variable_control: QWidget,
+    ) -> None:
+        value: bool | float | None = None
```

Comment:
I think we need to use the old style convention because we support python3.8

Reply:
`from __future__ import annotation`
takes care of that